### DISCUSSION TITLE
Can't render image on localhost

### DISCUSSION THREAD

**Discussion by GuysmoB** [resend/react-email#1292] [Q&A] [answered]
Hi,
I have a image to include in an email. I added it to react-email-starter/static folder and also to the public/images folder.
If I run email dev, I can see it but once I run my app on localhost and I send an email, it doesn't render anymore.
I use Nextjs.

![image](https://github.com/resend/react-email/assets/42407006/2b6b4e56-b998-44d9-824b-a92f6779f4c7)

Any idea ?

Thanks !

**Comment by gabrielmfern** [comment, marked as answer]
Because you are running from localhost, your `process.env.VERCEL_URL` is not being filled in for the image. You'll probably only be able to test this on emails if you make a preview deployment to Vercel as it will have that filled in.

On another note, it seems like you're using the older version of `react-email`. So for anyone coming across this, we have since changed the `static` directory to be inside your `emails` as of `2.0.0`, the `create-email` is still outdated on this, with the exception of canary, as of the date of posting this.

**Comment by GuysmoB** [comment]
Thank you for the response.

Unfortunately, I cannot upgrade to the latest version due to a bug with Next.js.
However, I resolved the issue by uploading the image to my storage provider and using its absolute path.

**Reply by gabrielmfern** [reply]
> Unfortunately, I cannot upgrade to the latest version due to a bug with Next.js.

What bug did you experience? Is there already an issue open for it here?
